awsService/src/main/java/com/kovan/app/service/S3Service.java

```diff
@@ -0,0 +1,226 @@
+package com.kovan.app.service;
+
+import com.kovan.entity.Document;
+import com.kovan.exception.FileException;
+import com.kovan.service.DocumentService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+import software.amazon.awssdk.core.ResponseInputStream;
+import software.amazon.awssdk.core.sync.RequestBody;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.model.*;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import static io.micrometer.common.util.StringUtils.isBlank;
+import static java.lang.Boolean.TRUE;
+import static java.util.UUID.randomUUID;
+
+@Service
+@Slf4j
+public class S3Service {
+
+    @Value("${bucketName}")
+    private String bucketName;
+
+    private final S3Client s3Client;
+    private final DocumentService service;
+
+    public S3Service(S3Client s3Client, DocumentService service) {
+        this.s3Client = s3Client;
+        this.service = service;
+    }
+    public String createBucket(String bucketName) {
+        try {
+            CreateBucketRequest createBucketRequest = CreateBucketRequest.builder()
+                    .bucket(bucketName)
+                    .build();
+
+            s3Client.createBucket(createBucketRequest);
+            return "Bucket created successfully: " + bucketName;
+        } catch (S3Exception e) {
+            return "Error creating bucket: " + e.getMessage();
+        }
+    }
+
+    public String deleteBucket(String bucketName) {
+        try {
+            ListObjectsRequest listObjectsRequest = ListObjectsRequest.builder()
+                    .bucket(bucketName)
+                    .build();
+
+            ListObjectsResponse listObjectsResponse = s3Client.listObjects(listObjectsRequest);
```

Comment:
one place you are using ListObjectsV2Response one place you are using ListObjectsResponse. Please make consistent to all places

Reply:
Using ListObjectsV2Response in all places.